### DISCUSSION TITLE
multiple redirect urls in argocd-cm

### DISCUSSION THREAD

**Discussion by krisdock** [argoproj/argo-helm#1901] [Q&A] [answered]
is it possible to specify multiple urls in the argocd-cm configmap?

```
configs:
  cm:
    url:
```

edit to add: i don't see this value referenced in the chart. just curious what it actually does.

**Comment by pdrastil** [comment, marked as answer]
Hi unfortunately no as the url is tied to dex / sso authentication. If you are about exposing Argo CD on multiple domains then it's possible via ingress, however only domain `configs.cm.url` will be able to handle SSO callbacks.

**Reply by krisdock** [reply]
thank you for the reply @pdrastil
